services/movie.py

```diff
@@ -22,12 +28,14 @@ def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
     movie_description: str,
     genres_ids: list = None,
     actors_ids: list = None,
-) -> Movie:
+) -> QuerySet:
```

Comment:
The return type of `create_movie` should be `Movie` instead of `QuerySet` since it returns a single movie instance.